doc/sphinx/Pacemaker_Explained/status.rst

```diff
@@ -1,280 +1,347 @@
+
+       id
+     - :ref:`text <text>`
+     - Node ID (identical to ``id`` of corresponding ``node`` element in the
+       ``configuration`` section)
+   * - .. node_state_uname:
+
+       .. index::
+          pair: node_state; uname
+
+       uname
+     - :ref:`text <text>`
+     - Node name (identical to ``uname`` of corresponding ``node`` element in the
+       ``configuration`` section)
+   * - .. node_state_in_ccm:
+
+       .. index::
+          pair: node_state; in_ccm
+
+       in_ccm
+     - :ref:`epoch time <epoch_time>` *(since 2.1.7; previously boolean)*
+     - If the node's controller is currently joined at the cluster layer, the
+       epoch time at which it did so (or 1 if the node is in the process of
+       leaving the cluster), otherwise 0 *(since 2.1.7; previously, it was
+       "true" or "false")*
+   * - .. node_state_crmd:
+
+       .. index::
+          pair: node_state; crmd
+
+       crmd
+     - :ref:`epoch time <epoch_time>` *(since 2.1.7; previously an enumeration)*
+     - If the node's controller is currently joined at the cluster messaging
+       layer, the epoch time at which it did so, otherwise 0 *(since 2.1.7;
+       previously, the value was either "online" or "offline")*
+   * - .. node_state_crm_debug_origin:
+
+       .. index::
+          pair: node_state; crm-debug-origin
+
+       crm-debug-origin
+     - :ref:`text <text>`
+     - Name of the source code function that recorded this ``node_state``
+       element (for debugging)
+   * - .. node_state_join:
+
+       .. index::
+          pair: node_state; join
+
+       join
+     - :ref:`enumeration <enumeration>`
+     - Current status of node's controller join sequence (and thus whether it
+       is eligible to run resources). Allowed values:
+
+       * ``down``: Not yet joined
+       * ``pending``: In the process of joining or leaving
+       * ``member``: Fully joined
+       * ``banned``: Rejected by DC
+   * - .. node_state_expected:
+
+       .. index::
+          pair: node_state; expected
+
+       expected
+     - :ref:`enumeration <enumeration>`
+     - What cluster expects ``join`` to be in the immediate future. Allowed
+       values are same as for ``join``.
+
+
+.. _transient_attributes:
+
+.. index::
+   pair: XML element; transient_attributes
+   single: node; transient attribute
+   single: node attribute; transient
+
 Transient Node Attributes
 #########################
    
-Like regular :ref:`node_attributes`, the name/value
-pairs listed in the ``transient_attributes`` section help to describe the
-node.  However they are forgotten by the cluster when the node goes offline.
-This can be useful, for instance, when you want a node to be in standby mode
-(not able to run resources) just until the next reboot.
-     
-In addition to any values the administrator sets, the cluster will
-also store information about failed resources here.
+The ``transient_attributes`` section specifies transient
```

Comment:
We should probably define "transient" in this context. The previous description mentioned that these attributes are forgotten when the node goes offline.

Reply:
"node attributes" here links to the original section defining the terms